run.sh

```
@@ -0,0 +1 @@
+/syzkaller/bin/syz-manager -config /syzkaller/qemu.cfg
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)